loc/ui/messages.js

```diff
@@ -298,10 +315,11 @@ export default {
     vi: 'Bạn đã bị đá ra khỏi Party $1.',
     zh: '你被 $1 從 Party 踢了。',
     sr: '$1 te je izbacio iz party-ja.',
-    it: 'Sei stato cacciato dal party da $2.',
+    it: 'Sei stato cacciato dal party da $1.',
```

Comment:
Calling this out for other reviewers to see. Looks like an unrelated-to-Czech Italian bugfix. `$2` to `$1` to align with the other strings. Nice.

Reply:
Thank you.